global-utils/src/main/kotlin/org/yapp/globalutils/annotation/NoLogging.kt

```
@@ -0,0 +1,5 @@
+package org.yapp.globalutils.annotation
+
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class NoLogging()
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)